reference/UiTPAS.v2.json

```diff
@@ -25,6 +25,11 @@
       "description": "Operations related to ticket sales"
     }
   ],
+  "x-stoplight": {
```

Comment:
Eventueel de models onderbrengen in hun eigen Tag?

Reply:
Jep ik zal het even aanpassen